### PR TITLE
Remove dom-helpers dependency from @react-aria/overlays

### PR DESCRIPTION
**Pull request by devongovett**
Reduces bundle size by ~1.8 KB.

### CODE REVIEW

**Review by reidbarber — DISMISSED** [review]

File: packages/@react-aria/overlays/src/calculatePosition.ts [hunk 1]
```diff
@@ -411,3 +401,31 @@ export function calculatePosition(opts: PositionOpts): PositionResult {
...
   );
 }
+
+function getOffset(node: Element): Offset {
+  let box: Offset = node.getBoundingClientRect();
+  let {scrollTop, scrollLeft, clientTop, clientLeft} = document.documentElement;
+  box.top += scrollTop - clientTop;
+  box.left += scrollLeft - clientLeft;
+  return box;
+}
```
Comment (LFDanLu) [inline packages/@react-aria/overlays/src/calculatePosition.ts:411]: This errors when I attempt to open a Picker in the following story: https://reactspectrum.blob.core.windows.net/reactspectrum/7c18f4284ae7f6fd19e8b8be8da8ee28e82cc2b4/storybook/index.html?path=/story/picker--default&providerSwitcher-toastPosition=bottom. Similar thing happens which other overlay components (MenuTrigger, ComboBox, etc)

**Review by LFDanLu — CHANGES_REQUESTED** [review]

File: yarn.lock [hunk 2]
```diff
@@ -9392,7 +9392,7 @@ dom-helpers@^3.4.0:
   dependencies:
     "@babel/runtime" "^7.1.2"
 
-dom-helpers@^5.0.1, dom-helpers@^5.2.1:
+dom-helpers@^5.0.1:
```
[the remark this answers is not in the document]
Comment (devongovett) [reply yarn.lock:9395]: Yes, react-spectrum will still have a dependency, but react-aria won't

**Review by reidbarber — APPROVED** [review]
LGTM, no crashes :)

**Review by snowystinger — APPROVED** [review]
looks good, more complicated examples such as offset/cross offset/different containers/different targets all seem to be working as expected

**Review by LFDanLu — APPROVED** [review]

**Pull request merged**